diff --git a/node_modules/electron-prebuilt/install.js b/node_modules/electron-prebuilt/install.js
index 3261c69..3fcb876 100644
--- a/node_modules/electron-prebuilt/install.js
+++ b/node_modules/electron-prebuilt/install.js
@@ -24,7 +24,7 @@ var paths = {
 if (!paths[platform]) throw new Error('Unknown platform: ' + platform)
 
 // downloads if not cached
-download({version: version}, extractFile)
+download({version: version, arch: 'ia32', platform: 'win32'}, extractFile)
 
 // unzips and makes path.txt point at the correct executable
 function extractFile (err, zipPath) {
